src/solvers/fdsbp_tree/fdsbp_1d.jl

```diff
@@ -105,13 +113,13 @@ function calc_volume_integral!(du, u,
   # Use the tensor product structure to compute the discrete derivatives of
   # the fluxes line-by-line and add them to `du` for each element.
   @threaded for element in eachelement(dg, cache)
+    f_minus_plus_element = f_minus_plus_threaded[Threads.threadid()]
     f_minus_element = f_minus_threaded[Threads.threadid()]
     f_plus_element = f_plus_threaded[Threads.threadid()]
     u_element = view(u_vectors, :, element)
 
     # x direction
-    @. f_minus_element = splitting(u_element, Val{:minus}(), 1, equations)
-    @. f_plus_element  = splitting(u_element, Val{:plus}(),  1, equations)
+    @. f_minus_plus_element = splitting(u_element, 1, equations)
     mul!(view(du_vectors, :, element), D_plus, view(f_minus_element, :),
          one(eltype(du)), one(eltype(du)))
     mul!(view(du_vectors, :, element), D_minus, view(f_plus_element, :),
```

Comment:
Related to comment above. I find this a little confusing. To my inexperienced Julia eyes it is not obvious what is going on here. It looks like the data in `f_minus_plus_element` is computed but never accessed. 

The plus/minus flux data is computed and then stored into the `f_minus_plus_element` cache. But then it can be (somehow) directly accessed via the auxiliary `f_minus_element` or `f_plus_element` caches when we apply the derivative matrices. This maybe needs a comment or two explaining why this works.

Reply:
Good point. I will add some comments.

Reply:
Done in[86c4906](https://github.com/trixi-framework/Trixi.jl/pull/1282/commits/86c490665c58acc9d21e3307f3cad51385c97222)

Reply:
Do you think the new comments are enough, @andrewwinters5000, or shall I expand them?

Reply:
Thanks! Yes, this comment is quite helpful.